### DISCUSSION TITLE
Παραδοτέο 3/Π2019021

### DISCUSSION THREAD

**Discussion by StavroulaAnatolaki** [courses-ionio/sw#1333] [Show and tell]
Εγκατέστησα το Artix Linux σε VirtualBox. Αυτη τη στιγμη αντιμετωπιζω ενα προβλημα με το asciinema και προσπαθω να το λυσω. Οποτε χρησιμοποιω  φωτογραφιες για τωρα.

![VirtualBox_Artix Linux_04_03_2023_22_10_50](https://user-images.githubusercontent.com/72880868/222926977-e8679f81-1082-49fb-bc73-ccbd1249d1df.png)

**Comment by p20sami** [comment]
@StavroulaAnatolaki Επειδή εγκατέστησα και εγω τα Artix και είχα το θέμα οτι δεν κατέβαζε το asciinema (έλεγε asciinema:target not found αν θυμάμαι καλά), αν αντιμετωπίζεις αυτό το πρόβλημα τότε δες τις οδηγίες εδώ πρώτα ->(https://wiki.artixlinux.org/Main/Repositories#Universe) εκεί που λέει για το universe και μετά εδώ->(https://wiki.artixlinux.org/Main/Repositories) για να μπορείς να κατεβάσεις πακέτα του Arch στα Artix. (Το πακέτο artix-archlinux-support το εγκαθιστάς κανονικά με sudo pacman -S και τα αρχεία που λέει να αλλάξεις τα ανοίγεις με sudo nano (η vim) και την διαδρομή του αρχείου που λέει). Μπορεί σε κάποιες στιγμές να λέει να τρέξεις μια εντολή του pacman αφού αλλάξεις το αρχείο που λέει για να συγχρονιστεί η αλλαγή.

**Comment by voltmaister** [comment]
Ο παραπάνω συνάδελφος έχει δίκιο για το τι πρέπει να κάνεις άλλα, η εγκατάσταση artix είναι με μειωμένη βαθμολογία αφότου γίνεται μέσω αυτόματης εγκατάστασης.

**Comment by p20sami** [comment]
@voltmaister Σε αυτό που αναφέρεσαι, εξαρτάται ποιο iso θα διαλέξεις. Πράγματι υπάρχει iso με αυτόματη εγκατάσταση αλλά υπάρχει και iso όπου η εγκατάσταση είναι χειροκίνητη όπως αν εγκαθιστούσαμε arch Linux για παράδειγμα που πιστεύω το τελευταίο είναι αποδεκτό πλήρως

**Reply by voltmaister** [reply]
Το θέμα είναι ότι ο καθηγητής δεν μπορεί να ξέρει ποιο από τα δυο χρησιμοποιείς εσύ και νομίζω ότι εκεί βρίσκεται το πρόβλημα με το Artix αλλιώς σας λειτουργικό είναι όντως μια πολύ καλή επιλογή.

**Comment by StavroulaAnatolaki** [comment]
Πιστεύετε ότι η εγκατάσταση του λογισμικού σε φλασάκι είναι καλύτερη επιλογή από το VirtualBox;

**Reply by Angeloth1** [reply]
εξαρτάτε από το τι σε βολεύει εσένα πάντα όταν λειτουργείς ένα σύστημα κοντά στο μέταλλο είναι πιο λειτουργικό όμως το vm είναι πιο εύκολο κατέ-με. Γνώμη μου αν δεν θες το λειτουργικό να το τρέχεις σε πολλά μηχανήματα αλλά μόνο σε ένα χρησιμοποίησε κάποιο vm της αρεσκείας σου

**Reply by NickLitharis** [reply]
Η εγκατάσταση σε VirtualBox είναι πιο εύκολη και πιο ασφαλής σε σχέση με το bare metal. Όπως ανέφερε και συνάδελφος παραπάνω όλα έχουν να κάνουν με τη δική σου εξοικείωση πάνω στη διαδικασία
